utils/plugins.js

```diff
@@ -0,0 +1,30 @@
+/* eslint no-console: [ 'error', { allow: [ 'error' ] } ] */
+
+/**
+ * Validates the namespaced id.
+ *
+ * A namespaced id should be formatted like this: "namespace/identifier".
+ * It can only contain lowercase alphanumeric characters and dashes.
+ *
+ * If a namespace id is invalid, it will print a console error, optionally using
+ * the name of the thing it identifies.
+ *
+ * @param {string} id Namespaced identifier.
+ * @param {string} [name] Plural name of what the id is for, defaults to "Namespaced identifiers".
```

Comment:
1. Misalignment of parameter descriptions
2. Inconsistency of square brackets applied to argument name

Reply:
I'm not sure what you are referring to with #2. Isn't that how all optional parameters are documented in Gutenberg?